### PR TITLE
Add shares breadcrumb

### PR DESCRIPTION
**Pull request by JanAckermann**
## Related Issue

- Fixes https://github.com/owncloud/web/issues/6937

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

**Review by pascalwengerter — APPROVED** [review]
Works like a charm

**Comment by ownclouders** [comment]
Results for <strong>oCISFiles1</strong> https://drone.owncloud.com/owncloud/web/25561/56/1
<details><summary>:boom: The acceptance tests failed on retry. Please find the screenshots inside ...</summary>

<p>


### webUIDeleteFilesFolders-deleteFilesFolders_feature-L244.png
  


</p></details>
:boom: The acceptance tests pipeline failed. The build has been cancelled.

File: packages/web-app-files/src/views/shares/SharedResource.vue [hunk 1]
```diff
@@ -169,6 +169,10 @@ export default defineComponent({
 
     breadcrumbs() {
       return concatBreadcrumbs(
+        {
+          text: this.$gettext('Shares'),
```
Comment (pascalwengerter) [inline packages/web-app-files/src/views/shares/SharedResource.vue:173]: Wait no, this can only happen if sharing jail is there (oCIS feature)!

Comment (JanAckermann) [reply packages/web-app-files/src/views/shares/SharedResource.vue:173]: Ok, will add a check later, just sitting on other stuff right now

Comment (kulmann) [reply packages/web-app-files/src/views/shares/SharedResource.vue:173]: Check not needed. The view only gets navigated to in ocis with sharin jail. Oc10 navigates into mounted share in "All files" view.

**Pull request merged**